systems/citybuilding.go

```diff
@@ -5,39 +5,46 @@ import (
 
 	"engo.io/ecs"
 	"engo.io/engo"
+	"engo.io/engo/common"
 )
 
-type MouseTracker struct {
+// ---------------------------------
+// Vars
+
+// CityAssets are the assets for the system
+var CityAssets = []string{"textures/city.png"}
```

Comment:
It has some good qualities to it, especially when used in the `main` file. Nice thinking @Sendoushi :smile: